packages/picasso/src/Calendar/types.ts

```diff
@@ -41,3 +37,11 @@ export interface CalendarProps {
   children?: ReactNode
   hasFooter?: boolean
 }
+
+export type CalendarDateRange = { start: Date; end: Date }
+
+export type DateOrDateRangeType = Date | DateRangeType
+
+export type DateRangeType = [Date, Date]
```

Comment:
NIT: Maybe we can add some labels to the tuple values, wdyt?
```suggestion
export type DateRangeType = [start: Date, end: Date]
```

Reply:
@augustobmoura Added, thanks!